src/Flex.php

```diff
@@ -629,14 +629,17 @@ private function fetchRecipes(): array
 
     private function initOptions(): Options
     {
+        $extra = $this->composer->getPackage()->getExtra();
+
         $options = array_merge([
             'bin-dir' => 'bin',
             'conf-dir' => 'conf',
             'config-dir' => 'config',
             'src-dir' => 'src',
             'var-dir' => 'var',
             'public-dir' => 'public',
-        ], $this->composer->getPackage()->getExtra());
+            'root-dir' => $extra['symfony']['root-dir'] ?? '.',
```

Comment:
How is `.` resolved when using it? Is it related to the `composer.json` file by composer or something?

Reply:
It's the same as using getenv()

Reply:
You mean `getcwd()` I guess 😄